debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -825,14 +825,18 @@ private static int validateMessageKeyColumnsField(Configuration config, Field fi
     }
 
     private static int validateHostname(Configuration config, Field field, ValidationOutput problems) {
-        String hostName = config.getString(field);
-        if (!Strings.isNullOrBlank(hostName)) {
-            if (!HOSTNAME_PATTERN.asPredicate().test(hostName)) {
-                problems.accept(field, hostName, hostName + " has invalid format (only the underscore, hyphen, dot and alphanumeric characters are allowed)");
-                return 1;
-            }
-        }
+        // YB Note: Bypassing this check.
+        LOGGER.info("YB: Bypassing hostname validation");
         return 0;
+
+        //String hostName = config.getString(field);
+        //if (!Strings.isNullOrBlank(hostName)) {
+        //    if (!HOSTNAME_PATTERN.asPredicate().test(hostName)) {
+        //        problems.accept(field, hostName, hostName + " has invalid format (only the underscore, hyphen, dot and alphanumeric characters are allowed)");
+        //        return 1;
+        //    }
+        //}
+        //return 0;
```

Comment:
use if (!YugabyteDBServer.isEnabled()) instead of commenting out

Reply:
This file is not in the Postgres connector module so the `isEnabled` method is not visible here.

Reply:
As discussed: I will change the method from `private` to `protected` and then override in our `PostgresConnectorConfig`.

For future work: a PR will be raised against the upstream so that when we merge future commits, we do not hit merge conflicts.